src/less/tooltip/base/tooltip.less

```diff
@@ -162,6 +162,58 @@ span.tourtip {
     width: @icon-width;
 }
 
+.dialog-infotip[role="dialog"] {
```

Comment:
We should move all of this `dialog-infotip` under the dialog module. Infotip will then have a dependency on the dialog module.